feature/src/main/java/com/sopt/anshim/feature/main/NavGraph.kt

```diff
@@ -19,7 +20,9 @@ fun NavGraph(navController: NavHostController) {
             )
         }
         composable<Screen.AddBook> {
-            AddBookScreen()
+            AddBookScreen(
+                navController = navController
```

Comment:
navController를 직접 줘서 각 스크린에서 화면 이동을 관리하다보면 네비게이션 로직이 분산되어서 이곳에서 처리해주는게 조금 더 좋을 수도 있을 것 같아요!! 
```suggestion
                navigateToHome = {
                    navController.navigate(Screen.Home)
                }
```

Reply:
코드 수정이 필요해서 나중에 회의할때 이야기해보아요!!